CONTRIBUTING.md

```diff
@@ -2,7 +2,13 @@
 
 ## Generate Protobuf Go Files
 
-Make sure you have Docker installed.
+1. Make sure `opamp-go/internal/opamp-spec` submodule files exist. You can do this
```

Comment:
Shall we point out the `ssh` requirement? see also https://stackoverflow.com/questions/6031494/git-submodules-and-ssh-access

Reply:
Done.